docs/source/build.rst

```diff
@@ -1,6 +1,6 @@
 .. _build:
 
-conda build Recipe Reference
+Conda build recipe reference
```

Comment:
I think there is a little bit of inconsistency between the pages. Happy to change everything to title case.